publishing/README.md

```diff
@@ -0,0 +1,8 @@
+# Publishing conventions
+
+Your Starcounter app is best enjoyed when it is blendable with other apps. For that it needs to follow some conventions.
+
+These conventions exist for two reasons:
+
+1. to prevent apps that run in the same database from going in conflict
```

Comment:
Rephrasing:
to prevent apps that run in the same database from going in conflict ->
to prevent conflicts between apps in the same database